src/mbgl/tile/tile_worker.hpp

```diff
@@ -32,6 +33,8 @@ class TileParseResultBuckets {
 public:
     TileData::State state = TileData::State::invalid;
     std::unordered_map<std::string, std::unique_ptr<Bucket>> buckets;
+    std::unique_ptr<FeatureIndex> featureIndex;
```

Comment:
is `TileParseResultData` ok? 3b251aa `Data` doesn't really mean anything I'm not seeing how `Success` is a good fit either